src/actions/vm.js

```diff
@@ -76,6 +76,15 @@ export function suspendVm ({ vmId }) {
   }
 }
 
+export function editVm ({ vm }) {
```

Comment:
@jniederm well it is not vmId its whole vm, ediyVm dialog is filled with data from this paylod